src/figuresClasses.ts

```diff
@@ -1,19 +1,98 @@
+enum Shape {
+  Triangle = 'triangle',
+  Circle = 'circle',
+  Rectangle = 'rectangle',
+}
+
+enum Color {
+  Red = 'red',
+  Green = 'green',
+  Blue = 'blue',
+}
+
 export interface Figure {
+  shape: Shape,
+  color: Color,
+  getArea(): number,
+}
+
+function round(area: number): number {
+  return Math.floor((area) * 100) / 100;
+}
 
+function findSide(...sides: number[]): void {
```

Comment:
Function name should represent action that is performing by it. From name `findSide` I expect that this function will return some `side` of figure, but instead - it returns nothing. Please use more clear names for your variables and functions